### ISSUE TITLE
Batch Normalization with DDP

### ISSUE THREAD

**Issue by msalhab96** [pytorch/pytorch#91872]
### 🐛 Describe the bug

when training a model with DDP setting, the model training stuck when I use nn.BatchNorm1d, or nn.SyncBatchNorm layers
when I remove them from the model the training works normally

### Versions

torch 1.13

**Comment by LeeHY-1** [comment]
Hello, I also meet the same problem. With SyncBatchNorm, the training stucks after several iterations in the first epoch. And the training can run normally without SyncBatchNorm, but the train loss is much larger than the valid loss. Could you please tell me how to solve it? Thanks.
